explorer/client/src/utils/static/mock_data.json

```diff
@@ -443,6 +443,58 @@
                     }
                 }
             }
+        },
+        {
+            "hI1m16cQkwErCDqbPyuV2ZtX4DZEel2lQTU1b1VJq0c=": {
```

Comment:
ooc how did you generate that? are these real values that one could verify (i.e., on signatures, obj_ref etc)?